File: src/net/eduard/api/command/api/ApiLoadWorldCommand.java
package net.eduard.api.command.api;

import org.bukkit.Bukkit;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.modules.Mine;
import net.eduard.api.lib.manager.CommandManager;

public class ApiLoadWorldCommand extends CommandManager {
	public ApiLoadWorldCommand() {
		super("loadworld", "carregarmundo", "ligarmundo");
		setUsage("/api loadworld <world>");
		setDescription("Carrega um mundo descarregado no servidor");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		if (args.length == 1) {
			sendUsage(sender);
		} else {
			if (Bukkit.getWorld(args[1]) == null) {
				Mine.loadWorld(args[1]);
				sender.sendMessage("§bEduardAPI §aVoce carregou o mundo §2" + args[1]);
			} else {
				sender.sendMessage("§bEduardAPI §aEste mundo já esta carregado");
			}
		}
		return true;
	}

}

File: src/main/java/net/eduard/api/command/api/ApiLoadWorldCommand.kt
package net.eduard.api.command.api

import net.eduard.api.lib.manager.CommandManager
import net.eduard.api.lib.modules.Mine
import org.bukkit.Bukkit
import org.bukkit.command.Command
import org.bukkit.command.CommandSender

class ApiLoadWorldCommand : CommandManager("loadworld", "carregarmundo", "ligarmundo") {

    init {
        usage = "/api loadworld <world>"
        description = "Carrega um mundo descarregado no servidor"
    }
    override fun command(sender: CommandSender, args: Array<String>) {
        if (args.isEmpty()) {
            sendUsage(sender)
            return
        }
        val worldName = args[0]
        if (Bukkit.getWorld(worldName) == null) {
            Mine.loadWorld(worldName)
            sender.sendMessage("§bEduardAPI §aVoce carregou o mundo §f$worldName")
        } else {
            sender.sendMessage("§bEduardAPI §aO mundo §f$worldName §cjá esta carregado")
        }
    }


}